public/static/docs/command-reference/get.md

```diff
@@ -60,7 +60,7 @@ name.
 
 - `--rev` - specific
   [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
-  (such as a branch name, a tag, or a commit hash) of the DVC repository to
+  (such as a branch name, a tag, or a commit hash) of the Git/DVC repository to
```

Comment:
This was already OK as "Git repository. Just pending to apply the same to `dvc import` cmd ref. Most changes apply to both docs please.

Reply:
p.s. "Get/DVC" seems a little simplistic, let's try to avoid it. A full phrase like `<abbr>DVC project</abbr> or Git repository` (which we already use in other places) is best. Thanks

Reply:
Leaving as Git repository, and changed import.md's --ref as well.